pkgs/by-name/mp/mphidflash/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  libusb-compat-0_1,
+}:
+
+stdenv.mkDerivation {
+  pname = "mphidflash";
+  version = "1.8-unstable-2020-05-05";
+
+  src = fetchFromGitHub {
+    owner = "AdamLaurie";
+    repo = "mphidflash";
+    rev = "d65bedd3f564a5d91b29e06b4a5885c32780acc5";
+    sha256 = "sha256-akQjkkbGxkurBifTZuI+iVs8O2i8MM0LgMUYztg5hzE=";
+  };
+
+  nativeBuildInputs = [ libusb-compat-0_1 ];
+
+  buildFlags = [ "mphidflash64" ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    cp binaries/mphidflash-1.8-linux-64 $out/bin/mphidflash
+
+    runHook postInstall
+  '';
+
+  meta = {
+    description = "'mphidflash' is a simple command-line tool for communicating with Microchips USB HID-Bootloader and downloading new firmware.";
```

```suggestion
    description = "Command-line tool for communicating with Microchips USB HID-Bootloader and downloading new firmware";
```
Not start with the package name.
Not end with a period (or any punctuation for that matter).